txpipe/utils/calibrators.py

```diff
@@ -327,9 +325,6 @@ def apply(self, g1, g2, c1, c2):
 
         c2: array or float
             Shear 2 additive bias component
-
-        subtract_mean: bool
-            whether to subtract the constant c term (default True)
         """
 
         g1 = (g1 / (2 * self.R) - c1)/ (1 + self.K)
```

Comment:
what is K again? isn't R the response already? why is there additional calibration?

Reply:
I've never quite got the difference. Emily may understand it!